docs/software/social.md

```diff
@@ -28,3 +24,227 @@
     To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+## FreshRSS - A self-hosted RSS feed aggregator
+
+FreshRSS is a self-hosted RSS feed aggregator.
+
+![DietPi social software FreshRSS](../assets/images/dietpi-software-social-freshrss.jpg){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    - URL = `<http://<your.IP>/freshrss`
+    - Username = `dietpi`
+    - Password = `dietpi`
+
+=== "Usage"
+
+    Please refer to the official online documentation for usage information:  
+    <https://freshrss.github.io/FreshRSS/en/users/02_First_steps.html>
+
+## phpBB - Free flat-forum bulletin board software solution
+
+If you always wanted your very own forum, phpBB is all you need.
+
+Also Installs:
+
+- Webserver
+
+![DietPi social software phpBB](../assets/images/dietpi-software-social-phpbb.png){: style="width:500px"}
+
+### Access Forums
+
+URL = `http://<your.IP>/phpBB3`  
+(from DietPi v6.33 on: `http://<your.IP>/phpbb`)
```

Comment:
I added the DietPi v6.33 info during dev stage. Now that it's released, let's only show the new `/phpbb` location here, same for a few cases below.